app/src/main/java/app/revanced/integrations/patches/components/ShortsFilter.java

```diff
@@ -55,6 +61,9 @@ boolean isFiltered(final String path, final String identifier, final byte[] prot
         if (reelChannelBar.check(path).isFiltered())
             if (this.pathFilterGroups.contains(path)) return true;
 
+        // Shorts info panel path appears outside of reelChannelBar path.
+        if (infoPanel.isEnabled() && infoPanel.check(path).isFiltered()) return true;
```

Comment:
IIrc `check` already checks for `isEnabled`.

Reply:
I believe there was a motivation behind this. I think I had to use `check` somewhere despite the filter being enabled. Either way, looks good as is.